modal_proto/api.proto

```diff
@@ -934,6 +937,12 @@ message Function {
   // to look at fine-grained placement constraints.
   bool _experimental_scheduler = 49;
   optional SchedulerPlacement scheduler_placement = 50;
+  reserved 52; // _experimental_scheduler_placement
+
+  bool is_class = 53;  // if "Function" is actually a class grouping multiple methods
```

Comment:
Yeah most proto renames (except rpc method names) should be doable without too much work, but perhaps best to do it in a followup set of PRs, since it would most likely require synced changes in several repos to not break CI